helm-charts/FATE/templates/python-spark.yaml

```diff
@@ -301,7 +305,7 @@ spec:
               subPath: logs
             - mountPath: /data/projects/fate/conf1/
               name: python-confs
-            - mountPath: /data/projects/spark-2.4.1-bin-hadoop2.7/conf/spark-defaults.conf
+            - mountPath: /data/projects/spark-2.4.1-bin-hadoop2.7/conf/spark-defaults-bak.conf
```

Comment:
why "-bak"?

Reply:
Same question from my side

Reply:
I changed the file name to be more semantic, `spark-default-template`.